src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/Hooks/useVedtaksbegrunnelser.ts

```diff
@@ -67,6 +67,7 @@ export const useVilkårBegrunnelser = ({
                     VedtakBegrunnelseType.EØS_OPPHØR,
                     VedtakBegrunnelseType.ETTER_ENDRET_UTBETALING,
                     VedtakBegrunnelseType.INSTITUSJON_OPPHØR,
+                    Vedtaksperiodetype.ENDRET_UTBETALING,
```

Comment:
Jeg skjønner ikke hvordan dette funker 🤯 Så vidt jeg kan se brukes bare `vedtaksperiodeTilVedtakBegrunnelseTyper` én gang og det er for å se om nøklene i `vedtaksbegrunnelseTekster.data` finnes i listen vi får fra funksjonen, men `vedtaksbegrunnelseTekster.data` sine nøkler er vel alltid av typen `VedtakBegrunnelseType`, så da vil de vel aldri være lik en Vedtaksperiodetype?

Reply:
Hvis jeg skal gjette, så blir de nok sammenlignet på string verdien koblet til enumene, og siden de er like, så går det an? 😂